Firmware/RTK_Surveyor/Form.ino

```diff
@@ -651,9 +696,19 @@ void createSettingsString(char* newSettings)
   }
   //stringRecord(newSettings, "activeProfiles", activeProfiles);
 
-  //System state at power on. Convert various system states to either Rover or Base.
-  int lastState = 0; //0 = Rover, 1 = Base
-  if (settings.lastState >= STATE_BASE_NOT_STARTED && settings.lastState <= STATE_BASE_FIXED_TRANSMITTING) lastState = 1;
+  //System state at power on. Convert various system states to either Rover or Base or NTP.
+  int lastState; //0 = Rover, 1 = Base, 2 = NTP
+  if (productVariant == REFERENCE_STATION)
+  {
+    lastState = 2; //Default to NTPSERVER. This is probably redundant? But just in case...
```

Comment:
Should default to base, no? Should we enum the states?

Reply:
I'm happy for it to default to Base instead of NTP. Let me know what you prefer. But we don't want it to default to Rover - which is the default for everything else...